buddyboss/buddyboss-platform
# PROD-875 Fix for admin to assign group parent

### Changes proposed in this Pull Request:



Fixes #1956 

### How to test the changes in this Pull Request:

1. Create at least two Admin accounts
2. Using the first admin you created, create multiple groups
3. Now logged in using the second admin account
4. Go to any groups that were created from the first admin account
5. Notice that nothing is on the list of Group Parent

### Proof Screenshots or Video


https://www.loom.com/share/f3d98f0fb4584f769cb78059bdad36cb
